### PR TITLE
Magicnumbers

### PR DESCRIPTION
**Pull request by BJSalvatore**
I am working to streamline the obstacle arrays. This will not work correctly yet, as I have not found the best way to duplicate each Obstacle object in the obstacleArray, resulting in only 10 obstacles, not 30. I am working on that now, but if you want to take a look at what I've done, but wait on merging it, maybe?

### CODE REVIEW

**Review by TheErikWolfe — CHANGES_REQUESTED** [review]
Pull the most recent version of develop branch and fix the merge conflicts before we review the pull request please!

**Comment by TheErikWolfe** [comment]
Also, change the branch you're trying to merge into to develop  instead of master!

File: main.js [hunk 1]
```diff
@@ -66,65 +81,51 @@ var homeSpaceArray = [0,0,0,0,0];
...
+    // Orignal code: new Obstacle(sprites, 6, 196, 120, 24, logstartpoint[8], 90, 120, 24, 'from left to right', 'medium', logbreakpoint[2])
 ];
 
-window.addEventListener('keydown',
-    function(event) {
-        if (lives > 0) {
-        var keypress = event.keyCode;
-        move(keypress);
-        }
-    })
```
Comment (dimitri16h) [inline main.js]: Where did this even listener go? ^

**Review by TheErikWolfe — APPROVED** [review]

**Comment by TheErikWolfe** [comment]
I have a couple comments:
* Those heights could probably all be set to 25 making the array of different heights just a single variable.
* the widths could be changed into multiples of 30 to make it even more simple

File: main.js [hunk 2]
```diff
@@ -40,19 +54,20 @@ var logstartpoint = [-87, -320, -553, -181, -508, -835, -120, -386, -652];
...
 var breakpoint = [-27, 470, -30, 470, -50, -33, -33];
 
-var startpoint1 = [440, -30, 440, -30, 440, 440, 440];
+var startpoint1 = [game.width, -30, game.width, -30, game.width, game.width, game.width];
 var startpoint2 = [613, -206, 616, -206, 636, 473, 473];
 var startpoint3 = [759, -382, 792, -382, 832, 506, 506];
 
+// moved desty , destwidth and destheight elements from Obstacle objects in obstacleArray
+// to own arrays for easier manipulation
+var desty = [490, 450, 410, 370, 330, 250, 210, 170, 130, 120];
```
Comment (TheErikWolfe) [inline main.js:63]: I have a couple comments:
* Those heights could probably all be set to 25 making the array of different heights just a single variable.
* the widths could be changed into multiples of 30 to make it even more simple

**Review by TheErikWolfe — APPROVED** [review]

**Review by kevin-moore-729 — APPROVED** [review]

File: main.js (hunk 1 above)
Comment (BJSalvatore) [reply main.js]: The eventListener is back in.

File: main.js (hunk 2 above)
Comment (BJSalvatore) [reply main.js:63]: Like this, Christian?
var destwidth = 25; // then i would remove indexing from destwidth in Obstacles.
var desty= [490, 460, 400, 370, 340, 260, 200, 170, 130, 100];

**Review by dimitri16h — APPROVED** [review]

**Pull request merged**